### ISSUE TITLE
Error when `drop_dir` on an empty directory

### ISSUE THREAD

**Issue by daattali** [karthik/rdrop2#31]
I just created a directory on dropbox and wanted to test the `drop_dir()` function. I get

```
Error in print.default(., n = n) : invalid 'na.print' specification
```

If I add a file the function works fine, when I remove the file and the folder is empty again the error comes back.

Similarly, if a non-existent directory is given, then the same error shows. It'd be nicer to get a more informative "path does not exist" error.

rdrop v0.6.5.99

**Comment by daattali** [comment]
I believe this can be closed with commit 9eeaa074def637bc768f0c527b3d1b5f325e38e2 ?

**Comment by karthik** [comment]
Yes! Thanks Dean.
